storage/flux/table.gen.go.tmpl

```diff
@@ -797,6 +778,141 @@ func (t *{{.name}}GroupTable) advance() bool {
 	return true
 }
 
+type {{.name}}AggregateMethod func([]int64, []{{.Type}}) (int64, {{.Type}})
+
+// determine{{.Name}}AggregateMethod returns the method for aggregating
+// returned points within the same group. The incoming points are the
+// ones returned for each series and the method returned here will
+// aggregate the aggregates.
+func determine{{.Name}}AggregateMethod(agg datatypes.Aggregate_AggregateType) ({{.name}}AggregateMethod, error){
+ 	switch agg {
+	case datatypes.AggregateTypeFirst:
+		return groupByFirst{{.Name}}, nil
+	case datatypes.AggregateTypeLast:
+		return groupByLast{{.Name}}, nil
+	case datatypes.AggregateTypeCount:
+		{{if eq .Name "Integer"}}
+		return groupByCount{{.Name}}, nil
```

Comment:
This is a little confusing, since `groupByCount` actually does a `sum`, as it should, but I had to double-check to make sure. Maybe just put a comment here explaining this?